pkg/service/pipeline.go

```diff
@@ -1811,6 +1811,10 @@ func (s *service) ListPipelineRuns(ctx context.Context, req *pipelinepb.ListPipe
 					return nil, fmt.Errorf("failed to load recipe metadata. pipeline UID: %s recipe reference ID: %s", run.PipelineUID.String(), run.RecipeSnapshot[0].Name)
 				}
 
+				err = s.converter.IncludeDetailInRecipe(ctx, "", dbRecipe, false)
```

Comment:
Hi @joremysh 
I found a run-logging bug when a pipeline includes an iterator. I've fixed it in this commit.